writers/writers.go

```diff
@@ -0,0 +1,17 @@
+package writers
+
+import (
+	"context"
+
+	"github.com/cloudquery/plugin-sdk/v4/message"
+)
+
+type Writer interface {
+	Write(ctx context.Context, res <-chan message.WriteMessage) error
+}
+
+const (
+	DefaultBatchTimeoutSeconds = 20
+	DefaultBatchSize           = 10000
```

Comment:
I guess those are not from this PR but I actually think those shouldn't exist and each plugin must define it's own because what would happen if we change the default - it would change on all plugins without us really being able to test what is the impact. cc @herman